insights/config/specs.py

```diff
@@ -216,6 +216,7 @@
     "modinfo"                   : CommandSpec("/usr/sbin/modinfo {module}", module=r"\S+"),
     "modprobe.conf"             : PatternSpec(r"etc/modprobe\.conf"),
     "modprobe.d"                : PatternSpec(r"etc/modprobe.d/.*\.conf"),
+    "mongodb_disk_space"        : SimpleFileSpec("mongodb_disk_space"),
```

Comment:
The simple file `mongodb_disk_space` and `postgres_disk_space` you required must not exist if you do not generate them explicitly. 
Are they generated by any collection tool (sosreport, spacewalk or foreman-debug)?
If there are such simple files, you write them like:
~~~
db_disk_space : All([SimpleFileSpec("mongodb_disk_space"),
                             SimpleFileSpec("postgres_disk_space")]),
~~~

But for `insights-data` (the archive collected by insights-client), I don't suggest you generate these files.
So, at last, you can write them like:

~~~
db_disk_space : All([CommandSpec("/bin/du -sh /var/lib/mongodb /var/lib/pgsql"),
                             SimpleFileSpec("mongodb_disk_space"),
                             SimpleFileSpec("postgres_disk_space")]),
~~~

Reply:
done.